### PR TITLE
Generics parsing & examples, builtins migration period

### PR DESCRIPTION
**Pull request by ntdesmond**

### CODE REVIEW

File: Docs/Specification/Chapter 2 - Types.md [hunk 1]
```diff
@@ -67,7 +67,8 @@ List is compound type, meaning it stores multiple values **of the same type** to
 
 ```java
 var lst: List<Integer> = []              // empty list of integers
-lst = [1, 2, 3]                          // list with 3 integers 
+lst = [1, 2, 3]                          // list with 3 integers
+lst.Append(1337)
 var last = lst[lst.Length().Minus(1)]    // access to last item
```
Comment (ntdesmond) [inline Docs/Specification/Chapter 2 - Types.md]: This line is fixed with PR #26

File: Examples/generics.ol [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        var pair = Pair(42, "Amogus")
+
+        list.Get(0).Set("sus", pair)
+        IO().WriteLine(list.Get(0).Search(pair))
+    end
+end
+
+class Pair<K, V> is
+    var key: K = null
+    var value: V = null
```
Comment (ntdesmond) [inline Examples/generics.ol:14]: I guess we need to think of `null` values.
There are none yet, I just added here to make the syntax analyzer consume it

**Review by DOCtorActoAntohich — DISMISSED** [review]
:trollface:

**Comment by DOCtorActoAntohich** [comment]
sh-

**Review by DOCtorActoAntohich — APPROVED** [review]
:trollface: (:trollface:)

**Pull request merged**